README.md

```diff
@@ -1,68 +1,69 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Hack Zero
+> Hackathon management web application <br>use in tandem with [Hack-Zero-Backend](https://github.com/hackthehack/hack-zero-v2-backend)
 
-## Available Scripts
+[![NPM Version][npm-image]][npm-url]
+[![Build Status][travis-image]][travis-url]
 
-In the project directory, you can run:
+Hack Zero is a serverless hackathon management system. It allows entities to create a single location to share
+and update participates on the details and happenings of a hackathon event.
+Participates are also able to sign up to gain access to a range of features.
 
-### `npm start`
+### Key Features
+ * User Registration
+ * User Login
+ * Hack Idea Creation
+ * Join Hack Idea Team
+ * Editing Hacks
```

Comment:
You might want to include a list of planned features (if we a defined one somewhere)

Reply:
Added one!